context_chat_backend/models/nc_texttotext.py

```diff
@@ -75,16 +76,42 @@ def _call(
 
         try:
             task = Response.model_validate(response).task
-            print(task)
+            print("Initial task schedule response:", task)
 
             i = 0
             # wait for 30 minutes
             while task.status != "STATUS_SUCCESSFUL" and task.status != "STATUS_FAILED" and i < 60 * 6:
-                time.sleep(5)
-                i += 1
-                response = nc.ocs("GET", f"/ocs/v1.php/taskprocessing/task/{task.id}")
+                if int(i/180) == 0:
```

Comment:
```suggestion
                if i < 60 * 3:
```

Would be slightly easier to grok, perhaps :)

Reply:
yeah sure, hehe